api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -119,6 +119,34 @@ message JwtProvider {
   //
   type.matcher.v3.StringMatcher subjects = 19;
 
+  // Requires that the credential contains an `expiration <https://tools.ietf.org/html/rfc7519#section-4.1.4>`_.
+  // For instance, this could implement JWT-SVID
+  // `expiration restrictions <https://github.com/spiffe/spiffe/blob/main/standards/JWT-SVID.md#33-expiration-time>`_.
+  // Unlike ``max_lifetime``, this only requires that expiration is present, where ``max_lifetime`` also checks the value.
+  //
+  // Example:
+  //
+  // .. code-block:: yaml
+  //
+  //     require_expiration: true
+  //
+  bool require_expiration = 20;
+
+  // Restrict the maximum remaining lifetime of a credential from the JwtProvider. Credential lifetime
+  // is the difference between the current time and the expiration of the credential. For instance,
+  // the following example will reject credentials that have a lifetime longer than 24 hours. If not set,
+  // expiration checking still occurs, but there is no limit on credential lifetime. If set, takes precedence
+  // over ``require_expiration``.
+  //
+  // Example:
+  //
+  // .. code-block:: yaml
+  //
+  //     max_lifetime:
+  //       seconds: 86400
+  //
+  google.protobuf.Duration max_lifetime = 21;
```

Comment:
looks great, thanks for making the change!

Reply:
Thanks for your comments, I agree, looks better now!